policy/auxTelMapper.yaml

```diff
@@ -0,0 +1,411 @@
+defects: ../description/defects
```

Comment:
There is quite a lot of duplication between this and what is already defined in `obs_base`.  I know it's a pain, but it would be really nice to remove as much of that duplication now as possible.  The cleaner the repo is to start, the easier it will be to keep cruft out.

Reply:
For now, I've just taken them all out, as they're not needed at all (as we don't have any on-sky data yet!). Have also added comments on what each item _is_.